Deploy-20231004

FXF48-Mila opened the pull request:
Adding information from field record for deploy 20231004

aditya-uw commented:
@ctuguinay I know Mila is still in the process of adding info but I think one of the failed tests say that 'MS' is an unrecognized initial for deployer/scribe. 'MS' is for Mila so could you add that into your test? Thanks! And stay tuned, let's try to get fully up-to-date this week on deployment PRs.

ctuguinay commented:
I changed the test so that your initials are accepted, but you'll still need to add the images for the tests to pass @FXF48-Mila

FXF48-Mila commented:
Yes, I will upload the images now.

FXF48-Mila commented:
@aditya-uw @ctuguinay Hi, could one of you guys help me check the PR for deploy-20231004?

aditya-uw commented on field_records/ubna_2023.csv:
```diff
@@ -108,7 +108,11 @@ Date and Time deployed (local) , Date and Time recovered (local) , AudioMoth # ,
+2023-09-28T10:54:00 , 2023-10-04T10:26:00 , L , 016 , E18 Bridge , 47° 39' 21.288'' N , 122° 17' 48.720'' W , 192000 , Medium , None , None , 4.173 , 3.620 , AK , CT , MS , recover-20231004 , Ikea Batteries Nighttime 0:00-15:00 UTC;
+2023-10-04T09:48:00 , (DATE-TIME) , A , 008 , Foliage , 47° 39' 25.638'' N , 122° 17' 29.442'' W , 192000 , Medium , None , None , 4.250 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 1:15-14:30 UTC; First day training Mila + Mollie;
+2023-10-04T09:48:00 , (DATE-TIME) , C , 001 , Central Pond , 47° 39' 17.412'' N , 122° 17' 34.578'' W , 192000 , Medium , None , None , 4.226 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 1:15-14:30 UTC; First day training Mila + Mollie;
+2023-10-04T09:47:00 , (DATE-TIME) , D , 009 , Telephone Field , 47° 39' 17.802'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.289 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 1:15-14:30 UTC; First day training Mila + Mollie;
```
> 2023-10-04T09:47:00 , (DATE-TIME) , D , 009 , Telephone Field , 47° 39' 17.802'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.289 , (VOLTAGE) , AK , CT , MS , (RECOVER-DATE) , Ikea Batteries Nighttime 1:15-14:30 UTC; First day training Mila + Mollie;

Hi @FXF48-Mila I think you may have forgot to update the lat/lon details for this row because its the same as Central Pond's row for the previous week,

> 2023-09-28T10:54:00 , 2023-10-04T10:55:00 , K , 002 , Central Pond , 47° 39' 17.802'' N , 122° 17' 34.140'' W , 192000 , Medium , None , None , 4.177 , 3.630 , AK , CT , MS , recover-20231004 , Ikea Batteries Nighttime 0:00-15:00 UTC;

Could you check again for this? Thanks!

aditya-uw commented:
@FXF48-Mila I like your "First day training Mila + Mollie" notes! Once you fix that lat/lon, you can squash and merge!

aditya-uw commented:
Ah @FXF48-Mila in that same row, you need a space after the `W`. We put a space before and after every comma `,` as you can see in the example. Add that space in when you can.

> D , 009 , Telephone Field , 47° 39' 25.638'' N , 122° 17' 29.442'' W, 192000 ,

FXF48-Mila commented:
@aditya-uw Hi Aditya, I have corrected the location information. However, the photo from Foliage on 20231004 doesn't have the location info from my phone, so I replaced it with the previously deployed location info. Is this ok?

aditya-uw commented:
> @aditya-uw Hi Aditya, I have corrected the location information. However, the photo from Foliage on 20231004 doesn't have the location info from my phone, so I replaced it with the previously deployed location info. Is this ok?

Yes that's fine!

aditya-uw commented:
I'll go ahead to squash and merge this because you made this PR yourself and I can be the "approver"! Good job!
